src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const SEARCH_URL = 'https://www.google.com/';
+
+export const LABELS = {
+	SEARCH: 'search',
```

Comment:
Even though we discussed that this Actor doesn't need to be as standardized, I will point out a few things to give you better muscle memory on it :)

We try to match key and value if we can
```suggestion
	SEARCH: 'SEARCH',
```